tests/azure/templates/pytest_tests.yml

```diff
@@ -0,0 +1,55 @@
+
+parameters:
+- name: build_number
+  type: string
+- name: scenario
+  type: string
+- name: ansible_version
+  type: string
+  default: ">=2.9,<2.10"
+- name: python_version
+  type: string
+  default: 3.6
+
+jobs:
+- job: Test_PyTests
+  displayName: Run pytests on ${{ parameters.scenario }}
+  timeoutInMinutes: 120
+  steps:
+  - task: UsePythonVersion@0
+    inputs:
+      versionSpec: '${{ parameters.python_version }}'
+
+  - script: |
+      pip install \
+        "molecule[docker]>=3" \
+        "ansible${{ parameters.ansible_version }}"
+    displayName: Install molecule and Ansible
+
+  - script: pip install -r tests/requirements.txt
+    displayName: Install other dependencies
+
+  - script: |
+      mkdir -p ~/.ansible/roles ~/.ansible/library ~/.ansible/module_utils
+      cp -a roles/* ~/.ansible/roles
+      cp -a plugins/modules/* ~/.ansible/library
+      cp -a plugins/module_utils/* ~/.ansible/module_utils
+      molecule create -s ${{ parameters.scenario }}
+    displayName: Setup test container
+
+  - script: |
+      pytest \
+        -m "not (playbook)" \
```

Comment:
This is different from the documentation, where no parentheses were used. One more reason to change it to `pytest` or `python`.